#!usr/bin/env python3
PACKAGE = "speed_eval_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("start_test", bool_t, 0, "Begin test period", False)
gen.add("max_speed", double_t, 0, "Max speed before penalty", 7, 0, 15)

exit(gen.generate(PACKAGE, "speed_eval", "SpeedEval"))
